paper-api/src/main/java/org/bukkit/inventory/RecipeChoice.java

```diff
@@ -79,10 +121,12 @@ public MaterialChoice(@NotNull Material... choices) {
          *
          * @param choices the tag
          */
+        @Deprecated
         public MaterialChoice(@NotNull Tag<Material> choices) {
             this(new ArrayList<>(java.util.Objects.requireNonNull(choices, "Cannot create a material choice with null tag").getValues())); // Paper - delegate to list ctor to make sure all checks are called
         }
 
+        @Deprecated
```

Comment:
Lets not deprecate since we don't have the alternative in, and Material isn't deprecated yet.